### DISCUSSION TITLE
flexoTHERM exclusive VWF 117/4 - CSV?

### DISCUSSION THREAD

**Discussion by AndreasExner** [john30/ebusd#1310] [Q&A]
Moin. 

Mir fehlen bei meiner Anlage ein par Daten. Ich bin mir aber nicht sicher, ob diese überhaupt auslesbar / schreibbar sind.

- flexoTHERM exclusive VWF 117/4 (brine)
- uniSTOR exclusive VIH RW 300/3 MR
- sensoCOMFORT VRC 720
- VR 71

Es fehlen mir Infos zum Umweltkreislauf (Sole), Aktueller Energieverbrauch sowie, und das ist in Kombination mit PV der Wichtigste Punkt, Einstellmöglichkeiten der Zusatzheizungen. 

Hat jemand schon ein "besseres" CSV für diese Anlage? Danke!

version: ebusd 23.2.p20230716
update check: version 23.3 available
device: 192.168.66.198:9999, enhanced, firmware 1.1[460f].1[460f]
access: "*"
signal: acquired
symbol rate: 62
max symbol rate: 167
min arbitration micros: 2
max arbitration micros: 87
min symbol latency: 6
max symbol latency: 48
scan: finished
reconnects: 0
masters: 5
messages: 516
conditional: 0
poll: 104
update: 10
address 03: master #11
address 08: slave #11, scanned "MF=Vaillant;ID=HMU00;SW=0308;HW=0403", loaded "vaillant/08.hmu.csv"
address 10: master #2
address 15: slave #2, scanned "MF=Vaillant;ID=CTLV2;SW=0514;HW=1104", loaded "vaillant/15.ctlv2.csv"
address 26: slave, scanned "MF=Vaillant;ID=VR_71;SW=0201;HW=0503", loaded "vaillant/26.vr_71.csv"
address 31: master #8, ebusd
address 36: slave #8, ebusd
address 71: master #9
address 76: slave #9, scanned "MF=Vaillant;ID=VWZ00;SW=0308;HW=0403"
address f1: master #10
address f6: slave #10, scanned "MF=Vaillant;ID=NETX3;SW=0121;HW=0404"

**Comment by akloud** [comment]
Ich habe eine flexoTHERM exclusive VWF 88/4 und würde gerne im Austausch bleiben. Im Moment arbeite ich mich noch in die Konfiguration ein.

**Comment by mwildbolz** [comment]
Hab selber eine VWF 117/4 - vielleicht findest du etwas Hilfreiches in [meinem Repository](https://github.com/mwildbolz/ebusd-configuration)

Lg

**Reply by akloud** [reply]
Deine letzte Änderung an 08.hmu habe ich übernommen. Danke!

Das Problem mit CTLV3 konnte ich mit einem symbolischen Link auf CTLV2 lösen:

> version: ebusd 24.1.24.1
> update check: OK, device firmware v5 1[4a1b] available
> device: /dev/ttyAMA0, serial high speed, enhanced, firmware 1.1[3501].1[3501]
> access: *
> signal: acquired
> symbol rate: 95
> max symbol rate: 176
> min arbitration micros: 2
> max arbitration micros: 19
> min symbol latency: 4
> max symbol latency: 5
> scan: finished
> reconnects: 0
> masters: 5
> messages: 475
> conditional: 0
> poll: 0
> update: 8
> address 03: master #11
> address 08: slave #11, scanned "MF=Vaillant;ID=HMU00;SW=0308;HW=0403"
> address 10: master #2
> address 15: slave #2, scanned "MF=Vaillant;ID=CTLV3;SW=0808;HW=8004", loaded "vaillant/15.ctlv3.csv"
> address 26: slave, scanned "MF=Vaillant;ID=VR_71;SW=0201;HW=0503", loaded "vaillant/26.vr_71.csv"
> address 31: master #8, ebusd
> address 36: slave #8, ebusd, scanning
> address 71: master #9
> address 76: slave #9, scanned "MF=Vaillant;ID=VWZ00;SW=0308;HW=0403"
> address ec: slave, scanned "MF=Vaillant;ID=SOL00;SW=0808;HW=8004"
> address f1: master #10
> address f6: slave #10, scanned "MF=Vaillant;ID=NETX3;SW=0124;HW=0404"

Gibt es noch eine einfache Verbesserung, die dir ins Auge sticht?

**Reply by akloud** [reply]
Morgen kommt Vaillant für die 5-Jahres-Garantie. Vielleicht bekomme ich da ja eine Info, was NETX3 und VWZ00 für Geräte sind.

**Reply by akloud** [reply]
Also ich bin mittlerweile wieder etwas lost:
Die neueste Version von ebusd (25.1) kann ich nicht nehmen, weil ich den ebus-Adapter im iobroker nutze, um meine Daten nach Grafana zu schreiben. Mit dem iobroker wollte ich dann z.B. auch die Speicherladung anstoßen. Das ist mir erst nach dem Update aufgefallen und ich habe jetzt die Version 24.1 installiert:

> version: ebusd 24.1.24.1
update check: version 25.1 available, device firmware v5 1[540f] available
device: /dev/ttyAMA0, serial high speed, enhanced, firmware 1.1[3501].1[3501]
access: *
signal: acquired
symbol rate: 103
max symbol rate: 215
min arbitration micros: 2
max arbitration micros: 896
min symbol latency: 0
max symbol latency: 15
scan: finished
reconnects: 0
masters: 5
messages: 762
conditional: 0
poll: 1
update: 29
address 03: master #11
address 08: slave #11, scanned "MF=Vaillant;ID=HMU00;SW=0308;HW=0403", loaded "vaillant/08.hmu.csv"
address 10: master #2
address 15: slave #2, scanned "MF=Vaillant;ID=CTLV3;SW=0808;HW=8004", loaded "vaillant/15.ctlv3.csv"
address 26: slave, scanned "MF=Vaillant;ID=VR_71;SW=0201;HW=0503", loaded "vaillant/26.vr_71.csv"
address 31: master #8, ebusd
address 36: slave #8, ebusd, scanning
address 71: master #9
address 76: slave #9, scanned "MF=Vaillant;ID=VWZ00;SW=0308;HW=0403"
address ec: slave, scanned "MF=Vaillant;ID=SOL00;SW=0808;HW=8004", loaded "vaillant/ec.sol.sc.csv"
address f1: master #10
address f6: slave #10, scanned "MF=Vaillant;ID=NETX3;SW=0124;HW=0404"

Ich nutze den ebusd auf einem Raspberry Pi mit folgenden Parametern:
EBUSD_OPTS="--scanconfig=full -d ens:/dev/ttyAMA0 --latency=50 --accesslevel=* -l /var/log/ebusd.log -p 8888 --httpport=8889 --htmlpath=/var/ebusd/html --enablehex -c https://cfg.ebusd.eu/ --pollinterval=1"

Sollte ich damit nicht den letzten, optimalen Stand der Konfiguration bekommen? Mit fehlen plötzlich Werte, die ich "früher" mit 
--configpath=/etc/ebusd/ebusd-configuration-master/ebusd-2.1.x/en/  
hatte. Zum Beispiel bekomme ich jetzt folgendes:
ebusctl read -c hmu CurrentCompressorUtil
ERR: invalid position in decode

**Comment by WulfgarW** [comment]
I am looking for ebusd configuration files that allow me to read the current power consumption of the immersion heater (additional heater) for a flexoTHERM VWF 118.
Has anyone of you already found the respective message definition for that?

**Reply by Redskyer** [reply]
I'm also interested in. If someone has optimized configuration, please let us know.

**Comment by nejcokorn** [comment]
I am also looking to get support for flexoCOMPACT exclusive VWF 58/4 with ebusd.

Correct me if I’m wrong, but it seems we are all in the same situation, since my HMU also reports HMU00;SW=0308;HW=0403. From what I found, the hardware version 0403 refers to the hardware plate, which appears to be the same as the one mentioned by the author of that post. I also have SW 0308, which I believe is the firmware running on that plate.

Has anyone made any progress with HMU00;SW=0308;HW=0403?

I have to admit I’m quite disappointed with Vaillant’s approach to system management and the amount of information that is available by default.

I live in a passive house, and the indoor temperature is often not quite right, so I constantly have to adjust the settings.

In summer, I use the heat pump for cooling. On several occasions the pump has simply stopped cooling even though the house was still too warm. The parameters in sensoComfort seem to hit edge cases, after which the system stops.

My goals are to:
Phase 1 - Retrieve basic information from the HMU.
Phase 2 - Actively monitor the HMU’s behaviour.
Phase 3 - Remove the CTLV2 (sensoComfort) controller and manage the HMU and CTLV2 directly myself.


On the other hand, RecoVair 260 is working perfectly in relation to ebusd. I will buy a micro-ebusd token, but I am not willing to buy one for flexoCOMPACT exclusive VWF 58/4 as it is not in the right shape yet.

**Reply by WulfgarW** [reply]
I made no progress, becauce the flexoTHERM is not mine. I asked for another user of the code I wrote. 
For my aroTHERM, the current power consumption of the immersion heater can be read from the VWZ00 (scan 76).

**Reply by nejcokorn** [reply]
Great, I will look into your changes made to the config files. Would you mind letting us know what is the HW and SW of your Vaillant machine.

**Reply by WulfgarW** [reply]
> Great, I will look into your changes made to the config files. Would you mind letting us know what is the HW and SW of your Vaillant machine.

Completely different from yours: HMU00, SW0522, HW5103

**Comment by woutersamaey** [comment]
Was someone able to get access to the:
- Brine circuit temp in
- Brine circuit temp out
- Brine circuit pressure
- Brine pump on/off


Thanks
